---
layout: default
title: Research
---

High-throughput technologies have generated a deluge of data describing living
organisms at unprecedented resolution. The wealth and diversity of biological
data available provide unique opportunities to understand the design principles
underpinning cellular life. The democratisation of DNA synthesis allow us to use
these design rules to engineer new designer systems ranging from genes to
chromosomes.

Our long term goal is to reverse-engineer biological systems to create
generative algorithms to design, build and test biological agents for addressing
healthcare and industrial biotechnology problems.



<div class="row">
    <div class="col-12">
        <h2>Synthetic genomics</h2>
    </div>
    <div class="col">
        <img src="/images/sc2-cover.jpg" width="90%" height="90%" class="rounded-circle">
    </div>
    <div class="col-9 mb-4">
        Recent advances in DNA synthesis, high-throughput sequencing and computer aided
        design (CAD) tools allow us to engineer the genomes of living cells and address 
        questions and tackle problems intractable with standard technologies. 
        We pioneered CAD software for synthetic genome engineering, which has been 
        instrumental to design the synthetic yeast genome 
        (<a href="http://www.syntheticyeast.org">Saccharomyces cerevisiae 2.0</a>, Sc2.0)
        the first synthetic eukaryotic genome ever built. 
        Sc2.0 allows us to address a number of open questions in genome biology, including 
        the identification of a minimal eukaryotic genome compatible with life.<br/>

        Synthetic chromosomes represent also a flexible chassis to integrate synthetic pathways
        into existing expression systems. However, the design principles to build new, 
        functional chromosomes are mostly unknown, and the current technology limits the DNA 
        molecules that can be synthesized. 
        We are addressing these issues by developing statistical models to learn how wild-type 
        genomes change upon the integration of synthetic chromosomes, and by developing methods
        to optimise manufacturing of chromosome scale molecules, by repositioning mathematical 
        programming methods we developed for electronic engineering.<br/>

        Our lab works in close collaboration with the <a href="http://www.genomefoundry.org">Edinburgh Genome Foundry</a>
        to scale-up our experimental work.
    </div>

    <!-- rare diseases -->
    <div class="col-12">
        <h2>Synthetic human enzyme engineering</h2>
    </div>
    <div class="col-9 mb-4">
        Enzymes are building blocks of cellular life and act as natural catalysts able 
        to accelerate almost any reaction. Enzymatic deficiencies are usually associated 
        with devastating rare diseases, which can only be treated by providing 
        the defective enzyme through intravenous injections. However, enzymes loose 
        catalytic activity in blood and often cause a severe immune response; moreover,
        current manufacturing technologies have low yield, which dramatically raises the cost of 
        treatment.<br/>

        Here we are building on our expertise in machine learning and synthetic genomics 
        to design and build human enzymes at scale. Our goal is to establish technologies 
        to optimise the therapeutic properties of synthetic enzymes and to engineer 
        expression systems for inexpensive production of these molecules, in order to 
        provide new sustainable treatment for patients with rare metabolic disorders.
    </div>
    <div class="col">
        <img src="/images/cell.jpg" width="90%">
    </div>

    <!-- cancer genetics and genomics -->
    <div class="col-12">
        <h2>Cancer genetics and genomics</h2>
    </div>
    <div class="col">
        <img src="images/hallmarks.png" width="100%"/>
    </div>
    <div class="col-9 mb-4">
        Decades of research have shown that genomic mutations in key genic regions 
        are responsible for the transformation of normal into cancer cells.
        However, while a causal role for somatic mutations has been shown for many 
        common malignancies, the role of high frequency inherited mutations has 
        remained elusive. 

        We are addressing this question by developing statistical learning methods
        to dissect the heritable risk of cancer at the gene level. 
        Nonetheless, the polygenic architecture of cancer requires linking gene level 
        information into pathways. 
        To do that, we are developing deep graph neural networks to integrate transcriptomic and proteomic 
        data and infer aberrant pathways involved in cancer metabolism and affecting response to therapy.<br/>

        While causal somatic mutations has been identified for many cancers, the genomic 
        landscape of rare tumours is mostly unknown. In collaboration with Blood Cancer Research Group 
        at University of Ostrava, we are completing the sequencing of the genome of rare blood cancers, such as
        multiple myeloma minimal residual disease and extramedullary myeloma. 
    </div>

    <!-- computational biology algorithms and software engineering -->
    <div class="col-12">
        <h2>Computational biology algorithms and software engineering</h2>
    </div>
    <div class="col-9">
    Computational methods are now cornerstone of many biological experiments. The
    lab is committed to release high-quality, open-source tools that can be
    easily integrated into analysis workflows. To do that, we adopt software
    engineering principles and methods that are standard in industry. Currently,
    our ecosystem relies on Python, Git, GitHub and GPU computing libraries. 
    All our analyses are implemented using either Nextflow or Snakemake
    workflow management systems. We also maintain a collection of Docker containers 
    to facilitate the adoption of our tools. You can check our growing suite of software on
    GitHub.
    </div>
    <div class="col">
        <img src="images/code.png" width="100%"/>
    </div>
</div>
